readme.txt

```diff
@@ -178,6 +178,10 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 == Changelog ==
 
+= [TBD] TBD =
+
+* Fix - Updated deprecated hook 'block_categories' to use 'block_categories_all'
```

Comment:
```suggestion
* Fix - Updated deprecated hook `block_categories` to use `block_categories_all`. [ET-1156]
```
We tend to use backticks so that the code is formatted when viewed in GitHub and other places. We also have to include the jira ticket id so that we can later track things.